### PR TITLE
New IPLD Video

### PR DESCRIPTION
**Pull request by mrodriguez3313**
I added the video into a place I thought would be a good spot.

### CODE REVIEW

File: content/en/curriculum/ipld/ipld-and-ipfs/index.md [hunk 1]
```diff
@@ -100,6 +99,9 @@ To achieve this you will need a **sharding** system that will break the data up
 More information about UnixFS can be found in the [**IPFS docs**](https://docs.ipfs.io/concepts/file-systems/#unix-file-system-unixfs) or in the UnixFS [**specification**](https://github.com/ipfs/specs/blob/master/UNIXFS.md).
 
 ## IPLD Data Types and Formats
+
+## Visualizing DAGs with IPLD | Marco Salinas - Fierro Labs
+{{% youtube 9bTEBDildIQ %}}
```
Comment (mrodriguez3313) [inline content/en/curriculum/ipld/ipld-and-ipfs/index.md:104]: I made an edit to the video, so I had to re-upload it.
```suggestion
{{% youtube Oh08rmAVeMQ %}}
```

**Review by walkerford — APPROVED** [review]

**Comment by mrodriguez3313** [comment]
@walkerford You can also check out the video here https://youtu.be/Oh08rmAVeMQ :thumbsup:

**Comment by walkerford** [comment]
The video still shows as private. @mrodriguez3313

**Comment by mrodriguez3313** [comment]
ty ty!! @kt-wawro

**Pull request merged**